Use the correct method for updating a column

felixclack opened the pull request:
Neo4j.rb doesn't implement the `set` method, which results in a
`NoMethodError` when using Carrierwave with a Neo4j enhanced model.

`write_attribute` does exist and seems the most appropriate choice for
ensuring the value of the uploaded file gets persisted.

tocker commented:
Hi @felixclack,

Thanks for PR!
Can you provide a reference code or test for this? I'd like to make sure this change is backward compatible.

-

felixclack commented:
I added a unit test for it but unfortunately it's pretty tightly coupled to the implementation. I guess it fulfills the need for documenting the intention of the change.

As far as I can tell, the method `write_attribute` has been available since version 2 of Neo4jrb. I'm unsure of when the `#set` method that I removed was introduced and then subsequently removed.

edenvicary commented:
@tocker and @felixclack any progress on this?

afestein commented:
Any further thoughts on this @felixclack? Our fork is now 7 commits behind cloudinary:master.

edenvicary commented:
@afestein looks like this is waiting on a response from Cloudinary?

tocker commented:
@felixclack thanks for the pull request! Sorry for the long delay...
